superset/dashboards/api.py

```diff
@@ -179,6 +179,9 @@ def ensure_thumbnails_enabled(self) -> Optional[Response]:
         "owners.id",
         "owners.first_name",
         "owners.last_name",
+        "owners.email",
+        "owners.id",
+        # "owners.user_attributes.avatar_url",
```

Comment:
@dpgaspar I think I might need your help to get this to work or to find some workaround. Here I need a handle on `avatar_url`  which I stored in the `UserAttributes` model, which is one-to-one with FAB's `User` model, and looks like there's some sweet magic here that's a bit beyond me - at least for now. 

I considered other approaches, like extending FAB's User model, but I don't think we have a handle on it in Superset RN. Other approach would be to support multiple dots here.

Curious on your thoughts